genie-web/src/main/java/com/netflix/genie/web/data/services/impl/jpa/JpaPersistenceServiceImpl.java

```diff
@@ -185,6 +185,16 @@
         .map(Enum::name)
         .collect(Collectors.toSet());
 
+    /**
+     * The a set containing statuses that come before CLAIMED.
+     */
+    @VisibleForTesting
+    static final Set<String> UNCLAIMED_STATUS_SET = Sets.newHashSet(
```

Comment:
Theres a few of these constants in here. Would it be better to locate them in the job status file?